turbopack/crates/turbo-tasks-fs/src/embed/dir.rs

```diff
@@ -69,8 +69,8 @@ macro_rules! embed_directory_internal {
         // make sure the types the `include_dir!` proc macro refers to are in scope
         use turbo_tasks_fs::embed::include_dir;
 
-        static dir: include_dir::Dir<'static> = turbo_tasks_fs::embed::include_dir!($path);
+        static DIR: include_dir::Dir<'static> = turbo_tasks_fs::embed::include_dir!($path);
```

Comment:
This was basically unrelated but i kept getting a warning from rust-analyzer about the namign here.